addons/core/functions/server/fnc_serverInit.sqf

```diff
@@ -113,6 +127,22 @@ GVAR(instanciationIsReady) = false;
             VARIABLE_DEFAULT(TFAR_freq_sr_west,true call DFUNC(generateSRSettings));
             VARIABLE_DEFAULT(TFAR_freq_sr_east,true call DFUNC(generateSRSettings));
             VARIABLE_DEFAULT(TFAR_freq_sr_independent,true call DFUNC(generateSRSettings));
+
+            if (TFAR_SameSRFrequenciesPerGroup) then {
+                [QGVAR(groupCreated), {
+                    params ["_group"];
+                    _frequencies = [TFAR_MAX_CHANNELS, TFAR_MAX_SW_FREQ, TFAR_MIN_SW_FREQ, TFAR_FREQ_ROUND_POWER] call TFAR_fnc_generateFrequencies;
+                    _group setVariable ["TFAR_freq_sr", _frequencies, true];
+                }] call CBA_fnc_addEventhandler;
+                
+                {
+                    _x params ["_unit"];
```

Comment:
_x is not an array. params doesn't really make sense if you are just doing `_unit = _x`

Reply:
seems like copy-paste leftovers. fixed